### ISSUE TITLE
Garage door sensor not detecting open/close status

### ISSUE THREAD

**Issue by milanmdev** [geerlingguy/pico-w-garage-door-sensor#10]
I've gone through and built my own garage door sensor based on the one explained in Jeff's video, but I've run into an issue.

When I try and close the garage door sensor, the LED does not turn off. I'm using the same parts described in the video (except for the breakout board, I had to order that directly from the seller [in China] so I'm waiting on that to arrive). The jumper cables are connected to the leads on the sensor (I twisted the leads on the sensor around the jumpers). Something weird I've experienced when running the `esphome run` command is that after the firmware is flashed and the Pi reboots, I get an `ERROR Could not connect to serial port /Volumes/RPI-RP2` error.

The firmware is still flashed though since when I ran the `led-blink` file it would blink, but still, show the same error.

Please let me know if I'm doing something wrong in my setup.

Screenshots:
The setup:
![image](https://i.imgur.com/Xlw6T1D.jpg)

The connections to the Pi:
![image](https://i.imgur.com/wnpaXzE.jpg)

`esphome run` command:
![image](https://user-images.githubusercontent.com/29466935/202852412-a88ad08a-fe35-4230-9f20-c8200318c6bf.png)

**Comment by milanmdev** [comment]
Nevermind, it seems that the LED wasn't functioning properly because the code has 2 garage doors but I'm only using one. I'm still a bit confused about the error for connecting to the serial port, but that is a separate issue so I'll open a new one soon (if I don't figure it out myself)
